eddymotion/estimator.py

```diff
@@ -143,6 +143,16 @@ def fit(
         return dwdata.em_affines
 
 
+def _to_nifti(data, affine, filename, clip=True):
+    data = np.squeeze(data)
+    if clip:
+        data = _advanced_clip(data)
+    nb.Nifti1Image(
+        data,
+        affine,
+        None,
+    ).to_filename(filename)
+
```

Comment:
Is this just changing the location of this function definition?

Reply:
Yes sorry it should not change anything. I had displaced it under a newly created utils folder, but it turns out I don't need this function in test anymore, so I put it back where it was originally.